src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,7 @@
   <data name="DelegateInvocationCanBeSimplified" xml:space="preserve">
     <value>Delegate invocation can be simplified.</value>
   </data>
-</root>
+  <data name="AddSwitchLabels" xml:space="preserve">
+    <value>Add Missing Labels</value>
```

Comment:
OK.  That reminds me--I think I need to remove the text from the language-specific configurations.